rustls/src/client/client_conn.rs

```diff
@@ -120,9 +120,9 @@ pub trait ResolvesClientCert: fmt::Debug + Send + Sync {
         sigschemes: &[SignatureScheme],
     ) -> Option<Arc<sign::CertifiedKey>>;
 
-    /// Return true if the client only supports raw public keys.  
-    ///  
-    /// See [RFC 7250](https://www.rfc-editor.org/rfc/rfc7250).  
+    /// Return true if the client only supports raw public keys.
```

Comment:
Hm. Could had sworn I've seen rustfmt fix trailing whitespace in comments before.

Reply:
nb. not objecting to fixing the whitespace here in passing, just confused how it got here in the first place.

Reply:
(I didn't even do anything to fix it, just set up VS Code to ditch trailing whitespace.)